velox/core/QueryCtx.h

```diff
@@ -133,8 +138,12 @@ class QueryCtx : public Context {
         std::make_shared<const MemConfig>(std::move(configOverrides)));
   }
 
+  folly::Executor* FOLLY_NULLABLE spillExecutor() const {
+    return spillExecutor_.get();
+  }
+
  private:
-  static Config* getEmptyConfig() {
+  static Config* FOLLY_NONNULL getEmptyConfig() {
```

Comment:
Would you extract FOLLY_NONNULL changes into a separate PR? This PR is super complex and having non-material changes makes reviewing even harder.